german/Smartphone/Internet/Internet.md

```diff
@@ -0,0 +1,8 @@
+Du bemerkst das das Booten des Smartphones so lange gedauert hat, das du mittlerweile bis zu den Knien im Wasser stehst. Das Ruderboot ist zum Glück aus Holz und geht nicht ganz unter. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.